jobs/director/spec

```diff
@@ -179,6 +179,14 @@ properties:
     description: RufusScheduler cron formatted schedule for cleanup of orphaned disks and orphaned snapshots
     default: '0 0,30 * * * * UTC' #every 30min
 
+  #Orphaned Networks automated cleanup
+  director.networks.max_orphaned_age_in_days:
+    description: Days to keep orphaned networks before cleanup
+    default: 5
```

Comment:
@jfmyers9 No, there is no specific reason for the 5 days default value. We were just trying to be consistent with the default value for orphaned disks which is 5 days. Should I change this to 1 day instead of 5?